<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlgUi</class>
 <widget class="QDialog" name="MainDlgUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magnetometer Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="MagnetometerWidget" name="widget"/>
     </item>
     <item>
      <widget class="QGroupBox" name="angles">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Angles</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="angle_x">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="angle_y">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Z</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="angle_z">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget1" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="shift_center">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Shift center</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="controls">
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Duration (Seconds):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_duration">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_comm">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnetometerWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>magnetometer_widget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edit_duration</tabstop>
  <tabstop>edit_comm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
